state-recover/clients/execution-layer-json-rpc-client/src/main/kotlin/build/linea/staterecover/clients/el/ExecutionLayerJsonRpcClient.kt

```diff
@@ -33,38 +40,65 @@ class ExecutionLayerJsonRpcClient internal constructor(
       }
   }
 
-  override fun lineaEngineImportBlocksFromBlob(blocks: List<BlockL1RecoveredData>): SafeFuture<Unit> {
+  override fun lineaEngineImportBlocksFromBlob(blocks: List<BlockFromL1RecoveredData>): SafeFuture<Unit> {
     return rpcClient
       .makeRequest(
-        method = "linea_engine_importBlocksFromBlob",
+        method = "linea_importBlocksFromBlob",
```

Comment:
Do we actually need json RPC state recovery API or in process API is sufficient?

Reply:
It was used when Gabriel was exposing this in a [Plugin here](https://github.com/Consensys/linea-state-recovery-plugin) before I migrated it to this to have just 1 plugin. 

Very likely will remove it. I just didn't do it because I was wondering if would be useful for testing to implement the JsonRpcExecutionLayerClient and run the StateRecoverApp as a stand alone. But likely I will remove this.